### ISSUE TITLE
GLT 2019 bookmarks.ics can not be imported in Google Calendar

### ISSUE THREAD

**Issue by jkirk** [linuxtage/glt-companion#23]
When trying to import the GLT bookmarks as ICS in a new Google Calendar I get the error:

  _Oops, we couldn't import this file. Please try again in a few minutes_

(And yes, I tried again after a few minutes - a few times).

What I did:
* Started GLT Companion (v1.7.1)
* Bookmarked a bunch of events
* In **Bookmarks** clicked the export button (:arrow_up:) and emailed me the `GLT 2019 bookmarks.ics` file
* On my desktop opened Google Calendar in Chromium (Version 73.0.3683.75 (Developer Build) built on Debian 9.8, running on Debian 9.8 (64-bit))
* Created a new Calendar ("GLT19- Bookmarks")
* Imported the exported file `GLT 2019 bookmarks.ics` to the new calendar "GLT19- Bookmarks". The error above is shown for a few seconds.

**Comment by Rufus125** [comment]
Hi!
Thanks for your bug reports.
Can you provide the ICS file or the events you bookmarked?
I just tried it and it worked perfectly fine for my ICS file.

**Comment by jkirk** [comment]
I do not have the ICS by the hand but will upload it as soon as I can.

Did you try a bookmark with multiple events? As far as I can remember, putting only one event in the bookmakr worked.

**Comment by Rufus125** [comment]
Yes, I tried with multiple events. It seems to be either that google calender ICS import was down for a short period of time, or only a specific event is affected.

**Comment by jkirk** [comment]
Thanks for your feedback!

I tried again, this time I manually removed all but one VEVENTs (BEGIN:VEVENT / END:VEVENT) and it failed again. I attach both ICS files:

[GLT19.zip](https://github.com/linuxtage/glt-companion/files/3170606/GLT19.zip)

Maybe you can provide the ICS file you used.

**Comment by Rufus125** [comment]
Hi!

I validated both ICS Files and they are valid just to be sure.
https://icalendar.org/validator.html
Then I imported them into a newly created calender here :
https://calendar.google.com/calendar/r/settings/export

1/1 and 21/21 successfully.
So no issues with these files on my side.
